src/modeler/AutoRest.Swagger/Validation/Extensions/LongRunningResponseValidation.cs

```diff
@@ -26,6 +36,6 @@ public class LongRunningResponseValidation : LongRunningExtensionRule
         /// <summary>
         /// The severity of this message (ie, debug/info/warning/error/fatal, etc)
         /// </summary>
-        public override Category Severity => Category.Warning;
+        public override Category Severity => Category.Error;
```

Comment:
sure I can run that. Are there any specific things you have in mind that I should be looking for as i run them?